docker/Dockerfile

```diff
@@ -18,28 +18,28 @@ FROM ${BASE_IMAGE}
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        autoconf \
-        autogen \
-        clangd \
-        gdb \
-        git-lfs \
-        libb64-dev \
-        libz-dev \
-        locales-all \
-        mosh \
-        openssh-server \
-        python3-dev \
-        rapidjson-dev \
-        sudo \
-        tmux \
-        unzip \
-        zstd \
-        zip \
-        zsh && \
+    autoconf \
+    autogen \
+    clangd \
+    gdb \
+    git-lfs \
+    libb64-dev \
+    libz-dev \
+    locales-all \
+    mosh \
+    openssh-server \
+    python3-dev \
+    rapidjson-dev \
+    sudo \
+    tmux \
+    unzip \
+    zstd \
+    zip \
+    zsh && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN pip3 install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cu118 torch==2.0.1+cu118 && \
+RUN pip3 install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cu118 torch==2.0.1 && \
```

Comment:
When you build without this change, you get an error that 2.0.1+cu118 doesn't exist

Reply:
weird, didn't see this error before